scripts/install.sh

```diff
@@ -1,26 +1,30 @@
 #!/bin/bash
 
-function install_bluemix_cli() {
-#statements
-echo "Installing Bluemix cli"
-curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github" | tar -zx
-sudo mv cf /usr/local/bin
-sudo curl -o /usr/share/bash-completion/completions/cf https://raw.githubusercontent.com/cloudfoundry/cli/master/ci/installers/completion/cf
-cf --version
-curl -L public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/Bluemix_CLI_0.5.4_amd64.tar.gz > Bluemix_CLI.tar.gz
+# This script is intended to be run by Travis CI. If running elsewhere, invoke
+# it with: TRAVIS_PULL_REQUEST=false [path to script]
+
+# shellcheck disable=SC1090
+source "$(dirname "$0")"/../scripts/resources.sh
+
+is_pull_request "$0"
+
+echo "Install Bluemix CLI"
+curl -L https://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/latest/Bluemix_CLI_amd64.tar.gz > Bluemix_CLI.tar.gz
 tar -xvf Bluemix_CLI.tar.gz
 sudo ./Bluemix_CLI/install_bluemix_cli
-}
 
-function bluemix_auth() {
-echo "Authenticating with Bluemix"
-echo "1" | bx login -a https://api.ng.bluemix.net -u $BLUEMIX_USER -p $BLUEMIX_PASS
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+echo "Install kubectl"
+curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/linux/amd64/kubectl
+chmod 0755 kubectl
+sudo mv kubectl /usr/local/bin
+
+echo "Install the Bluemix container-service plugin"
 bx plugin install container-service -r Bluemix
-echo "Installing kubectl"
-chmod +x ./kubectl
-sudo mv ./kubectl /usr/local/bin/kubectl
-}
+
+if [ -n "$DEBUG" ]; then
+  bx --version
+  bx plugin list
+fi
 
 function cluster_setup() {
```

Comment:
`function` is not needed in bash (it is still needed for bourne shell). We also no longer neef this function.